.github/2.0/cookbooks/Executor.md

```diff
@@ -514,5 +514,34 @@ class PaddleMwuExecutor(Executor):
             doc.embedding = np.array(_output)  # assign the encoding results to ``embedding``
 ```
 
+### PyTorch
 
+```python
+import torch  # 1.8.1
+import numpy as np
+
+
+from jina import Executor, requests
+
+
+class PytorchMwuExecutor(Executor):
+    def __init__(self, **kwargs):
+        super().__init__()
+        self.dims = 5
```

Comment:
u can remove this line and add it in the encoding_mat?

Reply:
I wanna keep it consistent with other cookbooks..